camera_matrix:
   rows: 3
   cols: 3
   data: [320.25492609007654, 0.0, 320.5, 0.0, 320.25492609007654, 240.5, 0.0, 0.0, 1.0]
distortion_coefficients:
   rows: 1
   cols: 5
   data: [0, 0, 0, 0, 0]
low_threshold_canny : 0
upper_threshold_canny : 20
publish_preprocess : true
publish_detected_platform : true
is_undistort : true
error_limit: 0.06
contour_perimeter_thresh: 250
contour_perimeter_scale: 25
kernel_size: 7
thresholding_parameters: 
   h:
      min: 0
      max: 255
   s:
      min: 0
      max: 255
   v:
      min: 0
      max: 20
